drivers/rpmsg/rpmsg_gpio.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * PRU Remote Processor Messaging Driver with gpiochip interface 
+ * For example codes to test and run the complete communication between PRU and ARM
+ * https://github.com/deebot/Beaglebone-BidirectionBus/tree/dev
+ * This software is licensed under the terms of the GNU General Public
```

Comment:
Give some space between documentation and license information for readability. Check out [SPDX](https://www.kernel.org/doc/html/v4.18/process/license-rules.html).